docs/configuration.md

```diff
@@ -663,24 +663,27 @@ Filters low-value coins which would not allow setting stoplosses.
 #### PriceFilter
 
 The `PriceFilter` allows filtering of pairs by price. Currently the following price filters are supported:
+
 * `min_price`
 * `max_price`
 * `low_price_ratio`
 
 The `min_price` setting removes pairs where the price is below the specified price. This is useful if you wish to avoid trading very low-priced pairs.
-This option is disabled by default, and will only apply if set to <> 0.
+This option is disabled by default (or set to 0), and will only apply if set to > 0.
 
 The `max_price` setting removes pairs where the price is above the specified price. This is useful if you wish to trade only low-priced pairs.
-This option is disabled by default, and will only apply if set to <> 0.
+This option is disabled by default (or set to 0), and will only apply if set to > 0.
 
 The `low_price_ratio` setting removes pairs where a raise of 1 price unit (pip) is above the `low_price_ratio` ratio.
-This option is disabled by default, and will only apply if set to <> 0.
+This option is disabled by default (or set to 0), and will only apply if set to > 0.
+
+For `PriceFiler` at least one of its `min_price`, `max_price` or `low_price_ratio` settings must be applied.
 
 Calculation example:
 
-Min price precision is 8 decimals. If price is 0.00000011 - one step would be 0.00000012 - which is almost 10% higher than the previous value.
+Min price precision for SHITCOIN/BTC is 8 decimals. If its price is 0.00000011 - one price step above would be 0.00000012, which is ~9% higher than the previous price value. You may filter out this pair by using PriceFilter with `low_price_ratio` set to 0.09 (9%) or with `min_price` set to 0.00000011, correspondingly.
 
-These pairs are dangerous since it may be impossible to place the desired stoploss - and often result in high losses.
+Low priced pairs are dangerous since they are often illiquid and it may also be impossible to place the desired stoploss, which can often result in high losses. Consider using PriceFilter with `low_price_ratio` set to a value which is less than the absolute value of your stoploss (for example, if your stoploss is -5% (-0.05), then the value for `low_price_ratio` can be 0.04 or even 0.02).
```

Comment:
@xmatthias pls review this advice separately (if I'm correct here)

Reply:
I don't think this is correct.

`PrecisionFilter` is the one that makes sure there's enough "room" below the current price to eventually place a stoploss (basically eliminating trades that trade at the lower end of their tradable range).


For a stoploss of -5% - you'll want to use 0.01 at least (which would mean it's 5 pips below entry).

If you set this value to 0.04, then your step size is 0.04 - so the stoploss can only trigger at -4% or -8% (no price in between is even possible due to the minimum pricemovement beeing so high) - so you'll raise your stoploss to -8% automatically (the first possible value below the configured stoploss).

Personally, i would not trade any pair that's got a minimum pip (minimum movement size) of > 0.001 (0.1%) - but i guess that depends on the strategy.

Reply:
I would again remove the "(or set to 0)" from the max_price description.

Also looks like a typo: PriceFiler instead of PriceFilter